packages/core/src/utils/createMap/pullPolarStyleToShadow.ts

```diff
@@ -10,7 +10,18 @@ export const pullPolarStyleToShadow = (
     const polarStylesheets = stylesheets.filter((el) =>
       el.getAttribute('data-vite-dev-id')
     )
-    polarStylesheets.forEach((style) => shadowRoot.appendChild(style))
+    // TODO: This is a temporary workaround until diplanung-style provides the rule this way; this should not find it's way onto main
```

Comment:
```suggestion
    // TODO: This is a temporary workaround until diplanung-style provides the rule this way; this should not find its way onto main, but it will, and, if you read it, probably did
```
Keeping it real and grammatically correct.